projects/igniteui-angular/src/lib/grids/columns/column.component.ts

```diff
@@ -1592,6 +1592,33 @@ export class IgxColumnComponent implements AfterContentInit, OnDestroy, ColumnTy
         return this._columnPipeArgs;
     }
 
+    /**
+     * @remarks
+     * Pass optional parameters to control properties of the default column editors.
+     * Accepts an `IColumnEditorOptions` object with the `dateTimeFormat` property.
+     * For more details see https://angular.io/api/common/DatePipe
+     * @example
+     * ```typescript
+     * const editorOptions: IColumnEditorOptions = {
+     *      dateTimeFormat: 'MM/dd/YYYY',
+     * }
+     * ```
+     * ```html
+     * <igx-column dataType="date" [editorOptions]="editorOptions"></igx-column>
+     * ```
+     * @memberof IgxColumnComponent
+     */
+    @notifyChanges()
+    @WatchColumnChanges()
+    @Input()
+    public set editorOptions(value: IColumnEditorOptions) {
+        this._editorOptions = value;
+        this.grid.pipeTrigger++;
```

Comment:
Assuming this is based on the `pipeArgs` setter, however I don't believe the `editorOptions` are used anywhere in the pipes, so triggering them is unnecessary extra work